lua/neogit/popups/help/actions.lua

```diff
@@ -15,9 +15,13 @@ local function present(commands)
       fn = fn[2]
     end
 
-    return { name = name, key = status_mappings[cmd], fn = fn }
+    return util.map(status_mappings[cmd] or {}, function(key)
+      return { name = name, key = key, fn = fn }
+    end)
```

Comment:
```suggestion
local keymap = status_mappings[cmd]
if keymap ~= nil then
  return { { name = name, key = table.concat(keymap, ","), fn = fn } }
end
```

That should show multiple keys for a single command instead of showing multiple commands with single keys.

Reply:
Yes you are correct.

It would create a table of `{ mapping, mapping, nil, mapping }`

... and of course lua being lua lists are defined as *continuous* integer keyed tables. Having a nil effecitely makes list operations only read up to the first nil, so the list is truncated. Much like null-terminated strings in C.

Have no idea who decided that a table is read `table[1], table[2], ...` until you encounter a nil, smh

Reply:
<img width="1070" alt="image" src="https://github.com/NeogitOrg/neogit/assets/25723553/136335dd-76b4-4e5f-8922-81848aca4ea3">

Reply:
My only bone to pick with using white space is that on the branch config popup there are multi key actions that are separated by whitespace. I think a comma is fine, just use a neutral color instead of the same color.